cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectS2IClusterTest.java

```diff
@@ -407,8 +410,9 @@ public void testGenerateDeploymentConfigWithTls() {
                 .endTls()
                 .endSpec()
                 .build();
-        KafkaConnectS2ICluster kc = KafkaConnectS2ICluster.fromCrd(resource, VERSIONS);
-        DeploymentConfig dep = kc.generateDeploymentConfig(Collections.EMPTY_MAP, true, null, null);
+        KafkaConnectS2ICluster kc = KafkaConnectS2ICluster.fromCrd(new Reconciliation("test", "kind", "namespace", "name"), resource, VERSIONS);
+        DeploymentConfig dep = kc.generateDeploymentConfig(new Reconciliation("test", "kind", "namespace", "name"),
```

Comment:
This should not need reconciliation if it is passed to the `KafkaConnectS2ICluster` instance.